core/controlplane/config/templates/cloud-config-controller

```diff
@@ -497,7 +497,7 @@ write_files:
       {{ end }}
 
       # Deployments
-      for manifest in {kube-dns-de,kube-dns-autoscaler-de,heapster-de}.yaml; do
+      for manifest in {kube-dns-de,kube-dns-autoscaler-de,heapster-de{{ if .ExportKubeResources }},cluster-dump{{ end }}}.yaml; do
```

Comment:
After thinking about this feature for several days, I'd suggest it to name `KubeResourcesAutosaveEnabled` rather than `ExportKubeResources` so that it can tell users that a save(export) happens automatically.